tasks/dependencies.rb

```diff
@@ -104,9 +104,9 @@ def berksfile_lock_task(task_name, dirs: [])
       # TODO in some edge cases, stable will actually be the latest chef because
       # promotion *moves* the package out of current into stable rather than
       # copying
-      puts "Getting latest chef 'current' version from omnitruck ..."
+      puts "Getting latest chef 'stable' version from omnitruck ..."
       options = {
-        channel: :current,
+        channel: :stable,
```

Comment:
We need a mechanism for developing against the `current` chef-client. There could be an environment variable here to change the `channel` option; other suggestions?